### ISSUE TITLE
Issues for label of Entry and Exit

### ISSUE THREAD

**Issue by xarthurx** [freqtrade/frequi#921]
**Is your feature request related to a problem? Please describe.**
The graph in the webUI only show "exit" point on the candles, but not "entry" point.

**Describe the solution you'd like**
Add illustration of "entry" point.

**Describe alternatives you've considered**
No.

**Additional context**
![image](https://user-images.githubusercontent.com/1921878/188569750-00150830-a5a2-407b-b47d-e0f0567f07ed.png)

**Comment by xmatthias** [comment]
I'm not entirely sure what you mean - entry points on the chart will be rendered (although your screenshot only contains exits on the chart).

so the result is - your strategy doesn't seem to generate entries (which would be green) for this timeperiod - or they're covered by exits.

we're intentionally not showing the long/short entries seperately to keep the legend somewhat legible if you use more than a few indicators.

**Comment by xarthurx** [comment]
![image](https://user-images.githubusercontent.com/1921878/188584747-d1617deb-aaaa-455b-952d-e86fe9c2b9b2.png)

I see, and realize that there are entry points.

The question is it seems there're more "exit points" than "entry points"... Would it be meaningful of the two matches in number?

**Comment by xmatthias** [comment]
there's a difference in what you see.

The green  and yellow entry/exit symbols are the signals your strategy generate (whenever enter_long/enter_short/exit_long/exit_short is set to 1).
your strategy will determine how many of these will be in any chart.
There's a ton of factors that can influence if a signal is acted on (creates a trade) or not. Best read the documentation for details on that.


Trades (the blue ones you have in this graph) are the real trades (when a position was entered / exited). the number of  trade entry / trade exits will match - unless the trade is still open - or the entry "aged out" of the chart.

**Comment by xarthurx** [comment]
I'd like to reopen this issue to a question related to this one.

![image](https://user-images.githubusercontent.com/1921878/188917386-b1263739-5d31-43f9-aa71-8fbe58342b8c.png)
I'm debugging a strategy and with this figure, it seems that the backtesting is not using the `max_open_trades` parameter from the config, or `  --max-open-trades INT`?

By zooming in:
![image](https://user-images.githubusercontent.com/1921878/188918011-08121ef0-e774-4620-abc8-0d20be0cbfbb.png)

There are multiple `enter_long` or `enter_short` consecutively, while I set the `max_open_trades = 1`.

The total number of signals generated also matches the diagram.


I tested the same strategy on TradingView, and compared the signals:
![image](https://user-images.githubusercontent.com/1921878/188918610-0c0932ec-6b9a-4172-accb-e4ca1b816b9b.png)

The timestamp that enters `long` or `short` in two systems matches.
However, `freqtrade` has more signals due to the issue I described above.

Is there any setting that I missed?
